source/slang/parser.cpp

```diff
@@ -4128,17 +4158,18 @@ namespace Slang
         // Add syntax for declaration keywords
     #define DECL(KEYWORD, CALLBACK) \
         addBuiltinSyntax<Decl>(session, scope, #KEYWORD, &CALLBACK)
-        DECL(typedef,       ParseTypeDef);
-        DECL(associatedtype,ParseAssocType);
-        DECL(cbuffer,       parseHLSLCBufferDecl);
-        DECL(tbuffer,       parseHLSLTBufferDecl);
-        DECL(__generic,     ParseGenericDecl);
-        DECL(__extension,   ParseExtensionDecl);
-        DECL(__init,        ParseConstructorDecl);
-        DECL(__subscript,   ParseSubscriptDecl);
-        DECL(interface,     parseInterfaceDecl);
-        DECL(syntax,        parseSyntaxDecl);
-        DECL(__import,      parseImportDecl);
+        DECL(typedef,         ParseTypeDef);
+        DECL(associatedtype,  ParseAssocType);
```

Comment:
Ditto for `ParseAssocType` -> `parseAssociatedType`. I know there's a lot of existing declarations that use the old convention, so this is a bit of a mess. If/when we have enough free cycles it would be good to do a cleanup pass over things and get them as uniform as possible.